core/src/com/biegan/game/Controller/AssetsMan.java

```diff
@@ -13,23 +13,37 @@ public AssetsMan() {
 
     public void load() {
```

Comment:
dalej nie to miałem na myśli. Chodziło mi, że assety powinny być wczytywane w odpowiednim momencie. Jeśli wszystkich assetów używasz w GameScreen, to spoko, ale chociaż nazwij tę funkcję loadGameScreenAssets a najlepiej mieć jakąś statyczną mapę, która dla każdego screena zwraca listę assetów do załadowania. Chodzi o to, żeby kod był jak najbardziej generyczny czyli 'reusable' to jest teraz sexy :D

Reply:
Tutaj nie wiem chyba jak to zrobić, myślę, że na spotkaniu jakoś to skumam, tej funkcji używam tylko raz w BieganBrickGame  `@Override
	public void create () {
		batch = new SpriteBatch();
		assetsMan = new AssetsMan();
		assetsMan.load();`